driver-examples/osgi/src/main/java/com/datastax/driver/osgi/impl/Activator.java

```diff
@@ -53,6 +79,17 @@ public void start(BundleContext context) throws Exception {
     public void stop(BundleContext context) throws Exception {
         if (cluster != null) {
             cluster.close();
+            /*
+            Allow Netty ThreadDeathWatcher to terminate;
+            unfortunately we can't explicitly call ThreadDeathWatcher.awaitInactivity()
+            because Netty could be shaded.
+            If this thread isn't terminated when this bundle is closed,
+            we could get exceptions such as this one:
+            Exception in thread "threadDeathWatcher-2-1" java.lang.NoClassDefFoundError: xxx
+            Caused by: java.lang.ClassNotFoundException: Unable to load class 'xxx' because the bundle wiring for xxx is no longer valid.
+            Although ugly, they are harmless and can be safely ignored.
+            */
+            Thread.sleep(10000);
```

Comment:
Sounds good, makes sense.  The exception doesn't really cause any particular issues right other than being logged?  I've encountered it in the past, but seems everything continues to work with it (just curious on its impact).